File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lightenup/domain/LightenUpLightbulbObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.lightenup.domain;

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState;

public class LightenUpLightbulbObject extends LightenUpObject {
    public AllowedObjectState state = AllowedObjectState.Normal;
    public String objTypeAsString() {
        return "lightbulb";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/lightenup/domain/LightenUpLightbulbObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.lightenup.domain

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState

class LightenUpLightbulbObject : LightenUpObject() {
    var state = AllowedObjectState.Normal
    override fun objTypeAsString(): String {
        return "lightbulb"
    }
}